### PR TITLE
text-utilities added

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**TEXT UTILITIES COMPONENT**

- Utilities added in the component :

1.  Heading.
2.  Text sizes.
3.  Text alignment.

Documentation (8)

### CODE REVIEW

File: style/components/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.txt-sm {
+  font-size: 0.8rem;
+  line-height: 1rem;
+  /* margin-top: 0.5rem;
+  margin-bottom: 1rem; */
```
Comment (devananth) [inline style/components/text-utilities.css:5]: These lines can be removed , if it is not needed anymore.

**Review by devananth — CHANGES_REQUESTED** [review]
Good work !

File: style/components/text-utilities.css (hunk 1 above)
Comment (tanishk15sharma) [reply style/components/text-utilities.css:5]: Thank you , changed the following .

**Review by MaruthiPothuganti — COMMENTED** [review]
Looks good to merge.  Defined small and meaningful class names.

**Review by darshandamre — COMMENTED** [review]
where are heading utilities? rest looks good.

**Review by Hardiegogo — APPROVED** [review]
Good to go.

**Review by spraveenofficial — APPROVED** [review]

**Review by rehanmohiuddin — APPROVED** [review]
Good To go

**Comment by tanishk15sharma** [comment]
> where are heading utilities? rest looks good.

it is above

File: docs.html [hunk 2]
```diff
@@ -894,6 +900,37 @@ <h1>Modals/popups are Awesome!</h1>
...
+            <h6>H6 - Heading</h6>
+            <div class="component-subhead">Text sizes</div>
+
+            <p class="txt-sm">This is small text.</p>
+
+            <p class="txt-md">This is medium text.</p>
+
+            <p class="txt-lg">This is large text.</p>
+            <p class="txt-xl">This is extra large text.</p>
+
```
Comment (agrawalshanu708) [inline docs.html:927]: use proper class name like text-small..

**Pull request merged**